configs/mae/benchmarks/vit-huge-p14_8xb128-coslr-50e_in1k_deepspeed-zero3.py

```diff
@@ -0,0 +1,144 @@
+_base_ = [
```

Comment:
better to inherit from file `vit-huge-p14_8xb128-coslr-50e_in1k` and remove unchanged keys